jina/parsers/orchestrate/runtimes/worker.py

```diff
@@ -92,4 +92,12 @@ def mixin_worker_runtime_parser(parser):
         nargs='*',
         help='List of exceptions that will cause the Executor to shut down.',
     )
+
+    gp.add_argument(
+        '--disable-reduce',
+        action='store_true',
+        default=False,
+        help='Disable the built-in reduction mechanism. Set this if the reduction is to be handled by the Executor itself by operating on a `docs_matrix`',
+    )
+
```

Comment:
why is this being configured at worker level now ?

Reply:
because this is a worker level parameter